environment/Dockerfile

```diff
@@ -52,7 +52,7 @@ RUN apt-get update -qq && apt-get install -qqy \
     xz-utils
 
 # Install Docker from Docker Inc. repositories.
-RUN curl -sSL https://get.docker.com/ | sh
+RUN curl -sSL https://get.docker.com/ | VERSION=5:19.03.2 sh
```

Comment:
Sorry this fell off my radar!

This actually brings up an interesting point - the docker host (a VM instantiated from the tailor slave template AMI) runs some version of docker, as provided by fedora. Automating the creation of that AMI is a big TODO.

I imagine we would want to match that version, though I don't know how compatibility between docker CLI and daemon works.